### ISSUE TITLE
Object detection, segmentation using elephas

### ISSUE THREAD

**Issue by riasto1** [maxpumperla/elephas#169]
Hi Elephas team, awesome project with great efforts, I manage to test it out with some simple model for classification problems. I wonder is there anyway for Elephas to be integrated in some popular net like YOLOv4, EfficientNet, etc. for object detection and segmentation problems. Could I have some documents or examples for them? Big thanks. I would also like to ask why there is a difference in accuracy using RDD and using DF, my friend test it and mention to me those differences. Thanks a lot.

**Comment by danielenricocahall** [comment]
Hello @riasto1 , I can add an example this week for object detection/segmentation in the `examples` folder. Also, regarding your second point - how big of a discrepancy was there in the accuracy? Is there a reproducible example? Thank you!

**Comment by riasto1** [comment]
Hello @danielenricocahall, really appreciate your support. First, about my second point in the first comment, my friend does the experience on malaria classification problem, I would like to ask and check if he still keeps the result. I would like to present more pressing points in my opinion, tf2 is currently rising in demand, and the support for tf1 will soon diminish. So I wonder if the team has a plan to cover the Elephas in tf2 as well, of course, this will be a tedious task, but I really wish for it. And tf2 already has support to a certain degree of distributed computing like parametric centralized server, load balance on all GPUs, etc. Has the team thought about integrated these with Elephas as well. Big thanks to you all

**Comment by danielenricocahall** [comment]
@riasto1 yes! TF 2 is officially supported as of Elephas 1.0.0 - please feel free to test out and play with the latest release (1.2.1). Regarding your second point, yes I know some distributed computing support has been introduced in TF 2.0 - we may try to incorporate some of the new features in future releases. Currently the team is just me, so it may take some time, but definitely stay tuned! :)

**Comment by riasto1** [comment]
That is very reassuring to hear that, thanks a lot, wish for the best to the team and the project
